pilot/prompts/development/feature_plan.prompt

```diff
@@ -0,0 +1,79 @@
+You are working in a software development agency and a project manager and software architect approach you telling you that you're assigned to add new feature to an existing project. You are working on a {{ app_type }} called "{{ name }}" and you need to create a detailed development plan so that developers can start developing the new feature.
+
+Here is a high level description of "{{ name }}":
+```
+{{ app_summary }}
+```
+
+{% if clarifications %}
+Here are some additional questions and answers to clarify the apps description:
+```
+{% for clarification in clarifications %}
+Q: {{ clarification.question }}
+A: {{ clarification.answer }}
+{% endfor %}
+```
+
+{% endif %}
+{% if user_stories %}
+Here are user stories that specify how users use "{{ name }}":
+```
+{% for story in user_stories %}
+- {{ story }}
+{% endfor %}
+```
+
+{% endif %}
+{% if user_tasks %}
+Here are user tasks that specify what users need to do to interact with "{{ name }}":
+```
+{% for task in user_tasks %}
+- {{ task }}
+{% endfor %}
+```
+
+{% endif %}
+Here are the technologies that you need to use for this project:
+```
+{% for tech in technologies %}
+- {{ tech }}
+{% endfor %}
+```
+
+Here are development tasks that specify what was already coded:
+```
+{{ development_tasks }}
+```
+
+{% if previous_features and previous_features|length > 0 %}
+After those tasks, new features were added to {{ app_type }} ""{{ name }}"". Here is a list of all features that are already implemented:
+{% for feature in previous_features %}
+- {{ loop.index }}. {{ feature['summary'] }}
+{% endfor %}
+
+{% endif %}
+Here is directory tree that shows current folder structure of project:
+```
+{{ directory_tree }}
+```
+
+App and all its features are already finished and working. Here are files that are currently implemented:
+{% for file in files %}
+**{{ file.path }}**:
+```
+{{ file.content }}
+```
+{% endfor %}
```

Comment:
We need to add a `function`/`tool` so that the LLM can request the files that it's interested - that's out of scope for this PR, but I think there's a lot of duplication in this file which will eventually need to be addressed in both locations